src/topology/continuous_function/bounded.lean

```diff
@@ -618,6 +628,16 @@ module.of_core $
 instance : normed_space 𝕜 (α →ᵇ β) := ⟨λ c f, norm_of_normed_group_le _
   (mul_nonneg (norm_nonneg _) (norm_nonneg _)) _⟩
 
+variables (𝕜)
+/-- The evaluation at a point, as a continuous linear map from `α →ᵇ β` to `β`. -/
+def eval_clm (x : α) : (α →ᵇ β) →L[𝕜] β :=
+{ to_fun := λ f, f x,
+  map_add' := λ f g, by simp only [pi.add_apply, coe_add],
+  map_smul' := λ c f, by simp only [coe_smul] }
```

Comment:
Tangential to this PR, but this raises to me that `coe_smul` has a weird statement, and should probably conclude `= c • f` instead of `= λ x, c • (f x)`.